module_intros/locality.Filter.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>What specific algorithmic issues? According to the docs, it controls "whether to&nbsp;exclude pairs of particles with the same point index in the output neighborlist (Default value =&nbsp;<code>True</code>)", and it seems like it's just a format thing? Even if there's an issue, should we just leave it at the doc's level so that it doesn't cause more confusion?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Yes, that's good point. I orignally thought both SANN and RAD had issues with division by 0, but it looks like it may only apply to RAD. I'll remove it from this notebook and we can talk more about it offline.</p>